### ISSUE TITLE
SD Card Config Files ?

### ISSUE THREAD

**Issue by bretdennison** [ctag-fh-kiel/ctag-straempler#12]
I flashed the ESP32 using the pre-built binaries.  It would appear I still need the config file(s) for the SD card?  I am unsure where to find this.

**Comment by ctag-fh-kiel** [comment]
Yeah, the ESP32 is picky with the type of SD cards, some are accepted some are not, some are accepted by read too slow for sample playback.
I have been using SanDisk 4GB cards quite successfully.
